woff1/tools/validator/validator.py

```diff
@@ -1877,7 +1877,7 @@ def write(self, text):
             self._elements[-1].text += text
 
     def compile(self, encoding="utf-8"):
-        f = StringIO()
+        f = BytesIO()
```

Comment:
`ElementTree.write` writes binary data, hence using a `BytesIO` is more appropriate (and required for Python 3).